src/mediapipe_internal/mediapipegraphdefinition.hpp

```diff
@@ -147,6 +153,8 @@ class MediapipeGraphDefinition {
     std::vector<std::string> inputSidePacketNames;
 
     std::atomic<uint64_t> requestsHandlesCounter = 0;
+
+    std::map<std::string, std::shared_ptr<PythonNodeResource>> pythonNodeResources;
```

Comment:
Don't use map if you do not care about sorted keys. I don't think we need it here

Reply:
Map was used in design so I suppose we will care about it.

Reply:
@mzegla ?

Reply:
Map was only highlighted as a high level data structure. Regarding specific implementation we should use the one that fits best. I don't think we will need ordering, so switching to unordered map sounds reasonable.